MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -115,34 +138,54 @@ void GeoMapper::mapData()
 		min_val = bounding_box.getMinPoint()[2];
 		max_val = bounding_box.getMaxPoint()[2];
 	}
-	std::size_t nPoints (points->size());
 
-	if (!is_borehole)
+	std::vector<GeoLib::Point*> const* points (_geo_objects.getPointVec(_geo_name));
+	std::size_t nPoints (points->size());
+	for (unsigned j=0; j<nPoints; ++j)
```

Comment:
`for (auto const* pnt : points)` would save two lines. ^^
Similar thing applies to the inner loop. But merely a question of taste.